src/modals/collectible-modal/components/collectible-attributes/collectible-attributes.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { FC } from 'react';
+import { StyleProp, View, ViewStyle } from 'react-native';
+
+import { styles } from './collectible-attributes.styles';
+import { CollectibleAttribute } from './components/collectible-attribute/collectible-attribute';
+
+interface Props {
+  attributes: {
+    attribute: {
+      id: number;
+      name: string;
+      value: string;
+      rarity?: number;
+    };
+  }[];
+  style?: StyleProp<ViewStyle>;
+}
+
+export const CollectibleAttributes: FC<Props> = ({ attributes, style }) => {
+  return (
+    <View style={[styles.root, style]}>
+      {attributes.map(({ attribute }, index) => (
```

Comment:
mb better to use Flatlist here?

Reply:
I think we don't need the FlatList here because don't have big data in this case